apps/condo/domains/user/schema/ConfirmPhoneAction.js

```diff
@@ -11,24 +11,53 @@ const { normalizePhone } = require('@condo/domains/common/utils/phone')
 const isEmpty = require('lodash/isEmpty')
 const { sendMessage } = require('@condo/domains/notification/utils/serverSchema')
 const { ConfirmPhoneAction: ConfirmPhoneActionGQL, generateSmsCode } = require('@condo/domains/user/utils/serverSchema')
+const { RedisGuard } = require('@condo/domains/common/utils/redisGuard')
+const { captchaCheck } = require('@condo/domains/common/utils/googleRecaptcha3')
```

Comment:
I think it's better to put it inside `user` domain. and if we have some other use cases we can move it inside common.

Reply:
Done